### PR TITLE
BFB-120: Crosshair

### PR DESCRIPTION
**Pull request by m4rtint**
**JIRA Ticket**
https://perigongames.atlassian.net/browse/BFB-120

**Description**
*Note* - There are more than a few ways to implement crosshair, after some googling, i'm going with the most simple one since I'm still experimenting with what works, what doesn't. 
Call this iteration 1
- Added few crosshairs
- Added script to set crosshair
- created prefab for crosshair

**Screenshots / GIFs**
image

### CODE REVIEW

File: Assets/_BForBoss/_Weapons/Scripts/CrosshairBehaviour.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        private void Start()
+        {
+            _crosshair.sprite = _defaultCrosshair;
+        }
+
+        [Button]
+        public void SetCrosshairImage(Sprite image)
+        {
+            _crosshair.sprite = image;
+        }
```
Comment (unoctium1) [inline Assets/_BForBoss/_Weapons/Scripts/CrosshairBehaviour.cs:26]: Is the point of this just so we can quickly test different sprites?

**Review by unoctium1 — APPROVED** [review]
lgtm, I'm not sure I get the point of CrosshairBehaviour but thats not blocking at all

File: Assets/_BForBoss/_Weapons/Scripts/CrosshairBehaviour.cs (hunk 1 above)
Comment (m4rtint) [reply Assets/_BForBoss/_Weapons/Scripts/CrosshairBehaviour.cs:26]: yup

**Comment by m4rtint** [comment]
> lgtm, I'm not sure I get the point of CrosshairBehaviour but thats not blocking at all

Originally, had the idea of having the crosshair dynamically enlarging and shrinking when you hold down the trigger.  And some other stuff, but that's a bit overcomplicating things, so just sticking with iteration 1 - simple first.

**Comment by unoctium1** [comment]
> Originally, had the idea of having the crosshair dynamically enlarging and shrinking when you hold down the trigger. And some other stuff, but that's a bit overcomplicating things, so just sticking with iteration 1 - simple first.

Sounds good, I think having the script in place will definitely make things better if we want to do anything more complicated later

**Pull request merged**